.github/workflows/pull-request-test.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           RENKU_ANONYMOUS_SESSIONS: true
           RENKU_RELEASE: ci-renku-${{ github.event.number }}
           RENKU_VALUES_FILE: "${{ github.workspace }}/values.yaml"
-          RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
+          RENKU_VALUES: ${{ secrets.CI_VALUES_FOR_TESTING_RESOURCE_SETTINGS }}
```

Comment:
Time to undo the change in this line and merge the PR 👍

Reply:
done

Reply:
of course now they refuse to run 🙈